app/helpers/buyers_helper.rb

```diff
@@ -0,0 +1,57 @@
+module BuyersHelper
+  def us_states
+    [
+      ['AK', 'AK'],
```

Comment:
I'm curious why this data structure is an array of arrays where the interior arrays have two elements with the same values?